.github/workflows/sync-cli-docs.yml

```diff
@@ -0,0 +1,89 @@
+name: sync-cli-docs
+
+on:
+  schedule:
+    # Run daily at 02:00 UTC
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "(optional) Docker CLI version - defaults to docker_ce_version in hugo.yaml"
+        required: false
+        default: ""
+  pull_request:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync-cli-docs:
+    runs-on: ubuntu-24.04
+    steps:
+      -
+        name: Checkout docs repo
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+      -
+        name: Get version from hugo.yaml
+        id: get-version
+        run: |
+          if [ -n "${{ inputs.version }}" ]; then
+            VERSION="${{ inputs.version }}"
+          else
+            VERSION=$(grep "docker_ce_version:" hugo.yaml | awk '{print $2}' | tr -d '"')
+          fi
+          # TODO(vvoland): Remove this after 29.2.0 is released
+          VERSION=60f06cb2df3df36ddfb531c1dae8c6fa96e5f9e7
```

Comment:
Gonna remove this after https://github.com/docker/cli/pull/6716 is included in the latest version